README.md

```diff
@@ -180,6 +180,15 @@ Mac:
 brew install pkg-config opus opusfile
 ```
 
+### Build tags
+
+This package can be built without depending on libopusfile by using the build tag
+`nolibopusfile`. For example:
+
+```sh
+go build -tags nolibopusfile ...
+```
```

Comment:
could you add a blurb here as to why anyone would ever want to do this? It's a pretty esoteric feature and it's good to know you can "safely ignore" it if you're not very experienced with the whole "dependencies, libraries, build systems" yet. Similarly, while the title is technically correct, it's quite generic. Could you reframe it as "Disabling libopusfile (for micro builds)" or something similar? Keep an audience in mind who might be overwhelmed by all the options, and who speaks poor English. It will help keep the issue count down :)

Reply:
in fact, the comment in your PR is a pretty good example of what I mean :)

Reply:
Sure thing!

Reply:
FWIW it works great on Alpine Linux, both static and dynamically linked to `libopus`. It even works on `scratch` with `musl` libc and `libopus*` copied from an Alpine builder image.